ctru-rs/src/gfx.rs

```diff
@@ -139,80 +182,128 @@ impl Gfx {
     }
 }
 
-impl TopScreen {
-    /// Enable or disable the 3D stereoscopic effect
-    pub fn set_3d_enabled(&mut self, enabled: bool) {
+impl<'screen> RawFrameBuffer<'screen> {
+    fn for_screen_side(screen: &'screen mut (impl Screen + ?Sized), side: Side) -> Self {
```

Comment:
Why not `&'screen mut dyn Screen`?

Reply:
Actually ended up just inlining this function definition into the one place it was used, avoiding the `?Sized` issues that cropped up otherwise

Reply:
Actually ended up just inlining this function definition into the one place it was used, avoiding the `?Sized` issues that cropped up otherwise